### DISCUSSION TITLE
Post what you do with python3-discogs-client!

### DISCUSSION THREAD

**Discussion by JOJ0** [joalla/discogs_client#39] [Show & Tell]
We would be very interested in what people code with python3-discogs-client. Please open a thread in this category of the discussion board. Also if you are just starting out with your projects, share it already!

Also I would see this section of the board as a place for just sharing project ideas and finding collaborators. Writing a software all alone feels good but on the long run is a very tedious job. Joining forces often is an option.

I'll make a start. This is DiscoDOS: https://github.com/joalla/discogs_client/discussions/41

**Comment by JOJ0** [comment]
I am specifically curious what @anssiahola and @vinylvault are about to code. You've been so active around here there must be something interesting in the queues already :-)

**Reply by AnssiAhola** [reply]
I'm making my thesis project, integrating the Discogs API into clients system so theres less manual work on discogs.com
I also did the PRs as a hobby and getting to know the API a bit better.

Kinda embarrassed to say that it looks like I might go with the php client instead as I managed to update it to reasonably modern version of php and the clients management system is based on php 😄 

Still I dont see these contributions as a waste, it was fun and I learned a lot.

Now that I think about it, I might do some personal project with this, not sure what tho 🤔 
Any ideas? 😄

**Reply by JOJ0** [reply]
If the client uses php, unfortunately you have to deal with it. You could suggest migrating everything to Python Wagtail/Django and it would only take you 1,5 years! But proooobably the client will say: "Whaaaat? No, you're fired"!

**Reply by VinylVault** [reply]
@JOJ0 I can certainly share the website once its finished, plenty more development needed though .... although you can check in on the progress **gone**

Be reasonably gentle, this is my first Python / Django project, although I have created the website a few times before in different languages ... Feedback is always welcome, and once the website is complete, I'm planning to break the Discogs only sections out into its own Django Module that people can add into websites of their own

**Reply by JOJ0** [reply]
nice site! I like the unusualness haha! great! also I think it's a great idea with the discogs django module. would def check it out since I got into django coding a little recently. wagtail cms actually

**Reply by VinylVault** [reply]
django discogs module? ive not used it

**Reply by JOJ0** [reply]
whaaat? LOL, you kidding right? you've just said you'll perhaps break out discogs stuff into a django module. I'd welcome that and try it out.

**Reply by VinylVault** [reply]
doh! misunderstood, thought you meant there was already one there

**Reply by VinylVault** [reply]
Well the site is finally live .... [https://www.thevinylvaultshow.co.uk](https://www.thevinylvaultshow.co.uk)
